app/src/main/java/tech/ula/model/repositories/AssetRepository.kt

```diff
@@ -59,6 +59,8 @@ class AssetRepository(
 
         val url = "$protocol://github.com/CypherpunkArmory/UserLAnd-Assets-" +
                 "$assetType/raw/master/assets/$architectureType/assets.txt"
+
+        if (!connectionUtility.httpsHostIsReachable("github.com")) throw object : Exception("Host is unreachable.") {}
```

Comment:
I see "github.com" being used many times, can we put this into strings.xml?

Reply:
strings.xml is meant more for user-facing strings as a list of things that require translation. It would be better to use a constant for sure, but this work will be part of the upcoming refactor anyway. I'm looking to just ship it for today so we can have it stress-tested over the weekend.